File: WebAPI/API/Models/Question/QuestionDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API.Models.Question
{
    public class QuestionDTO
    {

        public int Id;
        public int QuizId;
        public string Label;
        public QuestionType QuestionType;
        public int TimeLimit;
        public QuestionTrueFalse QuestionTrueFalse;
        public List<QuestionMultipleChoice> QuestionMultipleChoice;
        public List<QuestionAssociation> QuestionAssociation;
        public List<string> Categories;
        public bool NeedsAllAnswers;
        public int QuizIndex;
        
    }
}

File: Android/app/src/main/java/org/equipe4/quizplay/transfer/QuestionDTO.java
package org.equipe4.quizplay.transfer;

import android.widget.LinearLayout;

import java.io.Serializable;
import java.util.List;

public class QuestionDTO implements Serializable {

    public int id;
    public int quizId;
    public String label;
    public int questionType;
    public int timeLimit;
    public QuestionTrueFalse questionTrueFalse;
    public List<QuestionMultipleChoice> questionMultipleChoice;
    public List<QuestionAssociation> questionAssociation;
    public List<String> categories;
    public boolean needsAllAnswers;
    public int quizIndex;
}
